understanding-ibc-denoms/index.md

```diff
@@ -0,0 +1,400 @@
+
+Just like `samoleans` or `stake`, `ibc/27A639...` is the denomination (denom) of the token received from IBC, but `27A639...` is a hash of the denom, IBC port, and channel.
+
+Why is `27A639...` a hash? Paths that track the token on multiple hops from other blockchains to your account would be unbearably long. The Cosmos SDK has a 64-character limit on the denomination of the token.
+
+The tradeoff is that you must query the node to find out what the actual path and denomination is. This node query is called the `denomtrace`. 
+
+In Cosmos SDK, the Gaia daemon `gaiad` runs a full node of the gaia application and provides a gRPC interface. The default port is  9090. In this tutorial, you will learn to query this interface directly. 
+
+For now, follow along with the `gaiad` subcommands.
+
+```bash
+gaiad q ibc-transfer denom-trace 27A6394C3F9FF9C9DCF5DFFADF9BB5FE9A37C7E92B006199894CF1824DF9AC7C --node tcp://localhost:26557
+denom_trace:
+  base_denom: samoleans
+  path: transfer/channel-0
+```
+
+From this command, you now know that there is an IBC port `transfer` and channel `channel-0`. But to know the IBC light client behind the port and channel, you need to perform another query. 
+
+Why is it called a light client? Because it is a light client of the _other_ chain, keeping track of its blockhashes. The `ibc channel client-state transfer` command  explains the details of the denom path. 
+
+```
+gaiad q ibc channel client-state transfer channel-0 --node tcp://localhost:26557
+client_id: 07-tendermint-0
+client_state:
+  '@type': /ibc.lightclients.tendermint.v1.ClientState
+  allow_update_after_expiry: false
+  allow_update_after_misbehaviour: false
+  chain_id: ibc-0
+  frozen_height:
+    revision_height: "0"
+    revision_number: "0"
+  latest_height:
+    revision_height: "43"
+    revision_number: "0"
+  max_clock_drift: 600s
+  proof_specs:
+  - inner_spec:
+      child_order:
+      - 0
+      - 1
+      child_size: 33
+      empty_child: null
+      hash: SHA256
+      max_prefix_length: 12
+      min_prefix_length: 4
+    leaf_spec:
+      hash: SHA256
+      length: VAR_PROTO
+      prefix: AA==
+      prehash_key: NO_HASH
+      prehash_value: SHA256
+    max_depth: 0
+    min_depth: 0
+  - inner_spec:
+      child_order:
+      - 0
+      - 1
+      child_size: 32
+      empty_child: null
+      hash: SHA256
+      max_prefix_length: 1
+      min_prefix_length: 1
+    leaf_spec:
+      hash: SHA256
+      length: VAR_PROTO
+      prefix: AA==
+      prehash_key: NO_HASH
+      prehash_value: SHA256
+    max_depth: 0
+    min_depth: 0
+  trust_level:
+    denomination: "3"
+    numerator: "1"
+  trusting_period: 1209600s
+  unbonding_period: 1814400s
+  upgrade_path:
+  - upgrade
+  - upgradedIBCState
+```
+
+That's a lot of information, but it doesn't answer the question: how do you know if this IBC client can be relied upon?
+
+* Note the chain ID and the client ID.
```

Comment:
not sure what's going on here, but am very interested to learn and know more. Is the asterisk * on line 101 a way of denoting a subheading? or a way to signal one of a series of things that help us know if the IBC client is trusted?